Aleksandr_Mazurov/03.GIT.Hosting/03.GIT.Hosting.md

```diff
@@ -0,0 +1,12 @@
+[https://github.com/Mazurovsasha/03.Git.Hosting]
+
+[https://gitlab.com/Mazurovsasha/03.git.hosting]
+
+
+```bash
+
+#!/bin/bash
+
+git push -u origin --all
```

Comment:
please use loop here, you have duplicate code, list of upstreams should be collected by command and used inside loop